index.yaml

```diff
@@ -1,9 +1,29 @@
 apiVersion: v1
 entries:
   op-scim:
+  - apiVersion: v2
+    appVersion: v1.6.1
+    created: "2020-12-10T09:36:34.823167-08:00"
+    description: The 1Password SCIM bridge
+    digest: f5c6a69ecffd8b23d99278b75a38b9b7474b0e0eddc1143a87f38fa646827a73
+    home: https://support.1password.com/scim
+    icon: https://1password.com/img/logo-v1.svg
+    keywords:
+    - azure active directory
+    - okta
+    - provisioning
+    - SCIM
+    - 1Password
+    maintainers:
+    - email: support+business@1password.com
+      name: 1Password Platform Integrations Team
+    name: op-scim
+    urls:
+    - op-scim-1.6.1.tgz
+    version: 1.6.1
```

Comment:
ohhh even some optional stuff

Reply:
Yes! If I recall correctly, the index provides some nice package discovery mechanisms.